src/EFCore.MySql/Properties/MySqlStrings.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="IdentityBadType" xml:space="preserve">
-    <value>Identity value generation cannot be used for the property '{property}' on entity type '{entityType}' because the property type is '{propertyType}'. Identity value generation can only be used with signed integer, DateTime, and DateTimeOffset properties.</value>
+    <value>Identity value generation cannot be used for the property '{property}' on entity type '{entityType}' because the property type is '{propertyType}'. Identity value generation can only be used with integer, DateTime, and DateTimeOffset properties.</value>
```

Comment:
Am curious, does MySQL actually have unsigned integer types, and do those support AUTO_INCREMENT?

Reply:
Yes, it is even somehow encouraged (from [3.6.9 Using AUTO_INCREMENT](https://dev.mysql.com/doc/refman/8.0/en/example-auto-increment.html)):

> Use the smallest integer data type for the AUTO_INCREMENT column that is large enough to hold the maximum sequence value you will need. When the column reaches the upper limit of the data type, the next attempt to generate a sequence number fails. **Use the UNSIGNED attribute if possible to allow a greater range.**

Reply:
First time I hear of first-class unsigned type support in a relational database... Thanks for the info!